spec/functional/client_spec.rb

```diff
@@ -111,19 +111,6 @@
     expect(message.key).to eq "xoxo"
   end
 
-  example "delivering a message to a topic that doesn't yet exist" do
```

Comment:
This test was also removed from the upstream branch because the client should not throw an exception in this case, but a warning and continue. The consumer should continue to fetch cluster metadata and wait until the topic exists, and then poll it when it does.